weave

```diff
@@ -11,6 +11,18 @@ else
 fi
 IMAGE_VERSION=${VERSION:-$IMAGE_VERSION}
 
+# - Docker versions prior to 1.2.0 cannot parse `--dns-search=.`
+# - Docker versions prior to 1.3.0 fail to include the domain when
+#   setting the hostname, so lookups for non-fq names fail.
+# - Docker version 1.3.0 contains the severe bug
+#   https://github.com/docker/docker/issues/8632
+# - The weavexec image embeds a Docker 1.3.1 client. Misleading
+#   version mismatch errors are produced if the user is running with a
+#   client *and* server with a lower version.
+#
```

Comment:
Is there one more premise: that Docker will complain if you run a client against a server older than that client?